src/signals/IncidentMap/components/DrawerOverlay/styled.ts

```diff
@@ -179,6 +179,8 @@ export const CloseButton = styled(Button)`
   top: 14px;
   right: 20px;
   min-width: inherit;
+  // Needs z-index else content blocks the onClick
+  z-index: 1;
```

Comment:
We should consider moving indexes to a variables folder within a certain page, then we always know on what stack an element lives. I'll make a story.